### DISCUSSION TITLE
my project in idea have been mvn built in idea,but qodana is failed

### DISCUSSION THREAD

**Discussion by nyccccc** [JetBrains/Qodana#89] [Q&A]
my project has been mvn in the idea, without any problem. but when  I use the qondana-jvm, it throw a problem:Maven project XXX import problem: 
STRUCTURE:Could not find artifact

**Comment by brichbash** [comment]
Hello,

Could you please attach `idea.log` from `results` folder? (it is stored in `/data/results`, you can access it mounting your directory to the path, e.g. `docker run ... -v <your-directory>/:/data/results`)
